packages/ListBox/src/components/Options/helpers/options.js

```diff
@@ -201,16 +201,6 @@ export const handleClickOption = ({ props, state, dispatch, onChangeContext }) =
 };
 
 export function handleEnterOrSpace({ event, state, dispatch, onChangeContext }) {
-  const pressedSpaceKeyWhileHavingFilter = state.hasFilter && event.key === " " && event.target.value !== "";
```

Comment:
https://github.com/acl-services/paprika/pull/941 has the same change, will wait for that to get merged